### PR TITLE
[Scope change] Fix workorder link

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes:
[AB#211075](https://dev.azure.com/Equinor/fa63ceea-8883-41e2-8823-a3b54e4be178/_workitems/edit/211075)

## Checklist:

-   [x] I have performed a self-review of my own code.
-   [x] I have linked my DevOps task using the AB# tag.
-   [x] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/apps/ScopeChangeRequest/Components/WorkOrderTable/WorkOrderTable.tsx [hunk 1]
```diff
@@ -1,4 +1,8 @@
+import { tokens } from '@equinor/eds-tokens';
+import { isProduction } from '@equinor/portal-client';
 import { Column, Table } from '@equinor/Table';
+import styled from 'styled-components';
+import { useFacility } from '../../../../Core/Client/Hooks';
```
Comment (Noggling) [inline src/apps/ScopeChangeRequest/Components/WorkOrderTable/WorkOrderTable.tsx:5]: Alias :)

File: src/apps/ScopeChangeRequest/Components/WorkOrderTable/WorkOrderTable.tsx [hunk 2]
```diff
@@ -24,14 +30,34 @@ export function WorkOrderTable({ workOrders }: WorkOrderTableProps): JSX.Element
...
+        generateColumn(
+            'WO',
+            ({ workOrderNumber, workOrderId }) => (
+                <Link
+                    hideUnderline
+                    onClick={() => {
+                        console.log(workOrderId);
+                        window.open(
+                            //TEMP:
+                            `https://${isProduction() ? 'procosys' : 'procosys'
```
Comment (Noggling) [inline src/apps/ScopeChangeRequest/Components/WorkOrderTable/WorkOrderTable.tsx:47]: samme i test og prod?

Comment (Gustav-Eikaas) [reply src/apps/ScopeChangeRequest/Components/WorkOrderTable/WorkOrderTable.tsx:47]: Ja foreløpig, siden det kommer fra FAM som kun er PROD data

**Review by Noggling — APPROVED** [review]
OK

**Pull request merged**